docs/pages/extensions/meshmap/index.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: default
+title: MeshMap
+permalink: extensions/meshmap
+abstract: Collaborative cloud native visual designer to design and manage infrastructure and applications.
+language: en
+display-title: "false"
+list: include
+type: extensions
+---
+
+# <img style="height: 4rem; width: 4rem;" src="{{site.baseurl}}/assets/img/meshmap-icon-color.svg" /> MeshMap <sup style="font-size: 1.8rem; vertical-align: top;">BETA</sup>
+
+MeshMap is a GitOps-infused cloud native <b>visual designer</b> for Kubernetes and cloud native applications. MeshMap works with your Kubernetes clusters so that you design and manage your infrastructure and applications between members of your team. MeshMap has two modes: <a>Designer</a> and <a>Visualizer</a>. Create and share your own Kubernetes deployments and patterns in MeshMap Designer. Deploy your designs and manage your infrastructure using MeshMap Visualizer. It combines with feature of multi-user <a>collaboration</a> for cloud native infrastructure so you can team up to move design work forwared, faster. MeshMap removes all the challenging work of collaboration and gets everyone on the same page.
```

Comment:
Nitpik: 
In the fourth line, `you can team up to....` is it `forwared` or it should be `forwarded` or `forward` ? 
and can we also add `and` between `design work forward and faster`? What's your inputs? 🤔

Reply:
Thanks for your input, since I only changed the location of the file, I accidentally overlooked these issues. I rewrote the sentence. It is always helpful to have extra eyes to move forward.